frontend/post/postDetailsDirective.js

```diff
@@ -132,7 +132,10 @@
         };
 
         postDetailsCtrl.showCommentInput = function showCommentInput() {
```

Comment:
since you changed the body function, create js doc.

Reply:
Done